README.md

```diff
@@ -13,18 +13,20 @@ Job executions within Porter are not intended to be inspected directly or in rea
 Many input and output methods are supported to allow flexibility with other applications. For example, source files can come from HTTP or S3 endpoints, and callback messages can be sent via HTTP, [SNS](https://aws.amazon.com/sns/), and [SQS](https://aws.amazon.com/sqs/). The list of supported source and destination methods will grow over time; see below for a more complete list of methods that each aspect of the job execution support.
 
 ### Table of Contents
+
 - [Introduction](#porter)
 - [Execution Model](#execution-model)
 - [Messaging I/O](#messaging-io)
-    - [Starting a Job](#starting-a-job)
-    - [Input Message Format](#input-message-format)
-    - [Callback Messages](#callback-messages)
+  - [Starting a Job](#starting-a-job)
+  - [Input Message Format](#input-message-format)
+  - [Callback Messages](#callback-messages)
 - [Tasks](#tasks)
-    - [Inspect](#inspect)
-    - [Copy](#copy)
-    - [Image Transform](#image-transform)
-    - [Transcode](#transcode)
-    - [Transcribe](#transcribe)
+  - [Inspect](#inspect)
+  - [Copy](#copy)
+  - [Image Transform](#image-transform)
+  - [Transcode](#transcode)
+  - [Transcribe](#transcribe)
+  - [WAV Wrap](#wav-)
```

Comment:
Looks like this lost a bit, should be `#wav-wrap`

Reply:
fixed